### DISCUSSION TITLE
Can you add padding in specific directions for widgets?

### DISCUSSION THREAD

**Discussion by JardarIversen** [TomSchimansky/CustomTkinter#907] [General]
e.g. only above or on the right of widgets like in CSS. Would it be possible or has tkinter made it hard?

**Comment by TomSchimansky** [comment]
If you use the grid layout manager, you can set `padx=10` for the left and right side or `padx=(10, 20)` for left and right side individually. This works also for pady.

**Reply by JardarIversen** [reply]
Oh ok, and does it not work with pack? Also if i write `padx=10` how would it add padding to only one of the sides?

**Reply by Akascape** [reply]
@JardarIversen It also works with the pack method. Eg: `padx=(0,10)`, which means no padding on the left side but 10 on the right side.

**Reply by JardarIversen** [reply]
alright, thanks!
